packages/cubejs-schema-compiler/src/adapter/SnowflakeQuery.ts

```diff
@@ -11,7 +12,19 @@ const GRANULARITY_TO_INTERVAL = {
   year: 'YEAR'
 };
 
+class SnowflakeFilter extends BaseFilter {
+  public likeIgnoreCase(column: string, not: boolean, param: any, type: string) {
+    const p = (!type || type === 'contains' || type === 'ends') ? '%' : '';
+    const s = (!type || type === 'contains' || type === 'starts') ? '%' : '';
+    return `${column}${not ? ' NOT' : ''} ILIKE CONCAT('${p}', ${this.allocateParam(param)}, '${s}' ESCAPE '\\')`;
```

Comment:
We need to create an integration test in testing-drivers for such cases.

Reply:
Done!